types/react-simple-maps/tsconfig.json

```
@@ -1,7 +1,10 @@
 {
     "compilerOptions": {
         "module": "commonjs",
-        "lib": ["es6"],
+        "lib": [
+            "es6",
+            "DOM"
```

```suggestion
            "es6"
```
was the dependency on `DOM` required only due to `console.log` usage?

---

Not sure already, most probably yes. Let's check if tests would pass without `DOM`

---

ok, just confirmed, it's not due to `console.log`. Tests fail without this. see results here https://github.com/DefinitelyTyped/DefinitelyTyped/runs/4152944086?check_suite_focus=true

---

oh, yes, implicit dependnecy. Anyway, removing `console` would be nice, if you don't mind

---

yes, this is done already. `console.log` removed and replaced with `// $ExpectType` statements